client/src/components/Carousel.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+// import MobileStepper from '@mui/material/MobileStepper';
+import Typography from '@mui/material/Typography';
+import ExpandCircleDownOutlinedIcon from '@mui/icons-material/ExpandCircleDownOutlined';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+// import KeyboardArrowLeft from '@mui/icons-material/KeyboardArrowLeft';
+// import KeyboardArrowRight from '@mui/icons-material/KeyboardArrowRight';
+import makeStyles from '@mui/styles/makeStyles';
+import type { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  topBanner: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '1rem 0 1rem 0',
+  },
+  bannerLeft: {
+    display: 'flex',
+  },
+  bannerRight: {},
+  carouselContent: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  carouselWrapper: {
+    margin: 'auto',
+    width: '80%',
+  },
+  questions: {
+    margin: '0.75rem 0 0 3rem',
+    color: 'gray',
+  },
+}));
+
+type CarouselProps = {
+  steps: { label: string; imgPath: string }[];
+};
+
+function Carousel(props: CarouselProps) {
+  const classes = useStyles();
+  const [activeStep, setActiveStep] = React.useState<number>(0);
+  const maxSteps = props.steps.length;
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+  return (
+    <div className={classes.carouselWrapper}>
+      <div className={classes.topBanner}>
+        <div className={classes.bannerLeft}>
+          <Paper square elevation={0}>
+            <Typography sx={{ fontFamily: 'Poppins', fontSize: '2rem' }}>
+              Recent Needs From Nonprofits
```

Comment:
i guess this should be a prop really, sorry if my last comment on it was not clear
so parent goes `<Carousel label="Recent Needs From Nonprofits" ...`
and here we just use props.label or w/e